### ISSUE TITLE
Keypad emoji fail to render (sometimes)

### ISSUE THREAD

**Issue by gnprice** [zulip/zulip-mobile#3517]
We fixed #3129 a while ago, and the keypad emoji work great in a simulated Nexus 5 running Android 9 and a debug build from a recent master:

![image](https://user-images.githubusercontent.com/28173/59069488-1b74dd80-886c-11e9-8197-886edab1c19b.png)

But on my physical Pixel 2 XL, running also Android 9, with the recent release build v25.2.116, I see the symptoms of #3129 almost exactly: on the same messages from the screenshot above,
* the :one: and :hash: emoji in the messages are invisible (presumably the plain digits in white)
* the reaction with one :one: emoji renders as "1? 1", except I've written "?" here to stand for with an X-ed out rectangle (presumably a replacement glyph).

Perhaps a fonts issue? Note that as our unit tests show:
```
describe('nameToEmojiMap', () => {
  const check = (name, string1, string2) => {
    expect(string1).toEqual(string2);
    expect(nameToEmojiMap[name]).toEqual(string1);
  }; // ...

  test('works for some multi-codepoint emoji', () => {
    check('0',        '0⃣', '0\u{20e3}');
    check('asterisk', '*⃣', '*\u{20e3}');
    check('hash',     '#⃣', '#\u{20e3}');
  });
});
```
a keypad emoji like :hash: expands to the plain `#` character followed by U+20e3... and the latter is
```
$ unicode u+20e3
U+20E3 COMBINING ENCLOSING KEYCAP
```

so basically the font used on my phone might lack that combining character.

**Comment by gnprice** [comment]
Also works great on an iPhone 7 with iOS 12.3.1, running v25.4.118.

**Comment by gnprice** [comment]
Quick additional bit of data: the webapp in Chrome works great on the same messages and same device where the mobile app has the above symptom. That's my physical Pixel 2 XL running Android 9, with v25.6.120, the latest beta.

**Comment by gnprice** [comment]
@jainkuniya reported some additional information on #3537, a PR:
* if I understand correctly, he saw the issue also on a OnePlus 6, on Android 9.
* the problem is fixed by using a codepoint sequence like `0030-fe0f-20e3` rather than `0030-20e3`.

On further study, I believe the situation is that the two-codepoint sequence `0030-20e3` is just not a valid emoji sequence according to the spec, and it should be `0030-fe0f-20e3`. Details in a [subsequent chat thread](https://chat.zulip.org/#narrow/stream/3-backend/topic/unicode.20emojis/near/758828).

**Comment by gnprice** [comment]
We've had a new report of this:
https://chat.zulip.org/#narrow/stream/243-mobile-team/topic/number.20emoji/near/1167842

That chat thread on #backend back in 2019 didn't result in fixing these sequences on the server, so it's clearly past time we should just override them in the app as a workaround. I've just sent #4693 to do that.
